types/jest-matcher-utils/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for jest-matcher-utils 21.0
+// Project: http://facebook.github.io/jest/
+// Definitions by: Alex Coles <https://github.com/myabc>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as chalk from 'chalk';
+
+export const EXPECTED_COLOR: chalk.ChalkChain;
+export const RECEIVED_COLOR: chalk.ChalkChain;
+// export const EXPECTED_BG: chalk.ChalkChain; // removed in b430e51a
+// export const RECEIVED_BG: chalk.ChalkChain; // removed in b430e51a
+export const SUGGEST_TO_EQUAL: string;
+
+export function stringify(object: any, maxDepth?: number): string;
+
+export function highlightTrailingWhitespace(
+    text: string,
+    bgColor: chalk.ChalkChain // removed in b430e51a
```

Comment:
Also removed in https://github.com/facebook/jest/commit/b430e51a34b389c0f43dfdcf05fab5220a7b5677#diff-98e61ded296e35d3a4fcae207101edeb (PR https://github.com/facebook/jest/pull/4578). I'm not 100% sure if this is the right way to handle this.

Reply:
You can leave it in and always open a PR with an update once a new version without it is published - then there should be a published `@types` version both with and without it, so people on either version can get what they need.